custom-metrics-autoscaling/direct-to-sd/custom-metrics-sd-hpa.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Google Inc.
```

Comment:
this is not the correct application of the header, you need the full thing.

see https://github.com/GoogleCloudPlatform/freshpod/blob/master/yaml/dev/deployment.yaml

Reply:
Done

Reply:
Done